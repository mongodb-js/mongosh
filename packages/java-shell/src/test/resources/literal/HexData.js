// before
db.coll.insertOne({"_id": 1, v: new HexData(16, '31323334')});
// command checkResultClass
new HexData(16, '31323334')
// command checkResultClass
HexData(16, '31323334')
// command checkResultClass
db.coll.find().toArray()[0].v;
// clear
db.coll.drop();
