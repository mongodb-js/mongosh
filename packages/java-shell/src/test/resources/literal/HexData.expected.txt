BinaryResult: {"$binary": {"base64": "MTIzNA==", "subType": "10"}}
BinaryResult: {"$binary": {"base64": "MTIzNA==", "subType": "10"}}
BinaryResult: {"$binary": {"base64": "MTIzNA==", "subType": "10"}}
